packages/perl_yaml_tiny.rb

```diff
@@ -0,0 +1,32 @@
+require 'package'
+
+class Perl_yaml_tiny < Package
+  description 'YAML::Tiny - Read/Write YAML files with as little code as possible'
+  homepage 'https://metacpan.org/pod/YAML::Tiny'
+  version '1.63'
+  compatibility 'all'
+  source_url 'https://cpan.metacpan.org/authors/id/E/ET/ETHER/YAML-Tiny-1.73.tar.gz'
+  source_sha256 'bc315fa12e8f1e3ee5e2f430d90b708a5dc7e47c867dba8dce3a6b8fbe257744'
+
+  depends_on 'perl'
+
+  def self.build
```

Comment:
Same as `postinstall` and `check` part, if nothing need to be done in this part, just remove `def self.build` is fine